resource_sample_py/resource_sample/activities/db_activity.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+from temporalio import activity
+import os
+import psycopg2
+from psycopg2 import sql
+from common.messages import ComposeGreetingInput
+from psycopg2.extras import RealDictCursor
+
+
+@activity.defn(name="database_test")
+async def database_test(arg: ComposeGreetingInput) -> str:
+    db_config = {
```

Comment:
i guess we can pydantic this, and provide a singleton of connection from a centralized location (maybe shared libs...).